HIRS_Provisioner/src/main/resources/log4j2.xml

```diff
@@ -16,7 +16,7 @@
         </RollingFile>
     </Appenders>
     <Loggers>
-        <logger name="hirs" level="WARN" additivity="false">
+        <logger name="hirs" level="INFO" additivity="false">

```

Comment:
Considering that this would potentially enable logging of other INFO statements, perhaps the better idea would be to simply do a standard `System.out.println` with the usual `---->` prepended to the statement? That's how they seem to be handled in the corresponding shell scripts as `echo "-->Provisioning"`

What do you think @apldev4 or @apldev1

Reply:
Any thoughts?  I agree that changing the specific log.info() statements to System.out.println would probably be better in the long run than broadly changing the log level.

Reply:
Yeah I would suggest going ahead and making the change to `System.out.println`.